src/portable/options.rs

```diff
@@ -466,10 +454,35 @@ impl fmt::Display for StartConf {
 }
 
 pub fn instance_name_opt(name: &str) -> Result<(), String> {
-    if is_valid_name(&name) {
+    if is_valid_instance_name(&name) {
         return Ok(())
     }
     return Err("instance name must be a valid identifier, \
-                (regex: ^[a-zA-Z_][a-zA-Z_0-9]*$)".into())
+        regex: ^[a-zA-Z_][a-zA-Z_0-9]*$".into())
+}
+
+pub fn instance_or_cloud_name_opt(name: &str) -> Result<(), String> {
+    if is_valid_instance_name(&name) {
+        return Ok(())
+    }
+    if !name.contains("/") {
+        // todo: should this also mention the format for cloud instance names?
```

Comment:
Sure.

Reply:
I'm not sure how releases work in this project. Are we worried about hinting at cloud instance names in a pre 2.0 release of the CLI?

Reply:
or at least before the cloud is actually available?

Reply:
The upcoming cloud is not a secret, so it's fine to mention.